bits/shelx2cry/regressions_tests/references/wf5140.ins.out

```diff
@@ -2528,85 +2529,85 @@ CONT U[23]=-.00116, U[13]=.01434, U[12]=-.00424
 CONT RESIDUE=1
 CONT PART=8001
 #  C34D  1    0.109149    0.906680    0.173214   101.00000    0.04326    0.03921          0.03389    0.00261    0.00669   -0.00843
-ATOM TYPE=C, SERIAL=3441, OCC=.24845, FLAG=0
+ATOM TYPE=C, SERIAL=3441, OCC=1.00000, FLAG=0
```

Comment:
I missed that. Something is wrong here

Reply:
offending commit is:
https://github.com/ChemCryst/crystals/commit/65f80e48bdfff74effc9846ab87baa164fadc3ac

it is because of the branch if (atomslist(i)%sof < 100.0)
Any reason for this?